lib/model.js

```diff
@@ -176,62 +182,99 @@ Model.new = function(name, schema, options, thinky) {
   return model
 }
 
-Model.prototype.isReady = function() {
-  return this._getModel()._tableReady;
-}
+/**
+ * Create the model's table.
+ * @return {Promise=} Returns a promise which will resolve when the table is ready.
```

Comment:
Just wanted to say thank you for also writing these comments. I really appreciate your help :+1: